mempool/mempool.go

```diff
@@ -94,6 +98,12 @@ func NewMempool(config *cfg.MempoolConfig, proxyAppConn proxy.AppConnMempool) *M
 	return mempool
 }
 
+// FireOnTxsAvailable initializes the TxsAvailable channel,
+// ensuring it will trigger once every height when transactions are available.
+func (mem *Mempool) FireOnTxsAvailable() {
```

Comment:
Do we perhaps need to protect this from concurrent access, with a mutex? This is just a question from skimming and noticing that node uses this method.